sys/include/net/loramac.h

```diff
@@ -102,6 +102,13 @@ extern "C" {
  * @name    LoRaMAC default values
  * @{
  */
+/**
+ * @brief Default LoRaWAN region
+ */
+#ifndef LORAMAC_REGION_ACTIVE                  /**< To be used as Kconfig symbol */
```

Comment:
As discussed offline, this macro represents something like "define a symbol EU868 if this macro is defined". This would ensure that one symbol is defined.

But IMO this is only a hack for the case where the user doesn't use Kconfig in order to ensure that there's always one symbol defined. Kconfig will always define at least one symbol.

For the non Kconfig case the only thing we need is to ensure that a default value is selected IF there's no symbol selected. In practice the user would simply define a region or leave the defaults there. Why would a user need to define "LORAMAC_REGION_ACTIVE" instead of simply defining EU868? (or skipping any configuration?)

As proposed below, I think this should be the way to go:
```
#if !IS_ACTIVE(LORAMAC_REGION_EU868) \
    && !IS_ACTIVE(LORAMAC_REGION_IN865 \
    && !IS_ACTIVE(LORAMAC_REGION_EU433) \
    ...
#define LORAMAC_REGION_EU868)
#endif
```

There are only 10 regions and they won't change in the short term. There's no code duplication and this list doesn't need special maintainance.

Reply:
Considering non-Kconfig users, this makes sense.

Reply:
> Considering non-Kconfig users, this makes sense.

What makes sense? the original solution or the list?

Reply:
I meant the list you suggested makes sense for a non-Kconfig user

Reply:
ah, got it